5.6/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -80,10 +95,12 @@ EOSQL
   fi
 
   if [ -v MYSQL_ROOT_PASSWORD ]; then
+    log_info "Setting password for root ..."
```

Comment:
"Setting password for MySQL root user ..." otherwise this message will be pretty scary